.github/workflows/deploy-main.yml

```diff
@@ -13,81 +13,81 @@ permissions:
 
 # Allow one concurrent deployment
 concurrency:
-  group: "deploy-main"
+  group: 'deploy-main'
   cancel-in-progress: true
 
 jobs:
   build-distribution:
     runs-on: ubuntu-latest
     steps:
-      - name: "🛰️ Checkout source code"
+      - name: '🛰️ Checkout source code'
         uses: actions/checkout@v4
 
-      - name: "🛰️ Setup Node"
+      - name: '🛰️ Setup Node'
         uses: actions/setup-node@v4
         with:
-          node-version: "18.x"
-          registry-url: "https://registry.npmjs.org"
-          scope: "@kaoto"
-          cache: "yarn"
+          node-version: '20.x'
```

Comment:
the engine version is marked as 18.x https://github.com/KaotoIO/kaoto/blob/b201f6b3852a0d920a5ef252eccd496638a9244a/package.json#L34

Reply:
Yes, but the rest of the workflows are using `20.x`.